### PR TITLE
Update field_specials.c to match the style

### PR DESCRIPTION
**Pull request by sphericalice**
## **Discord contact info**
Spherical Ice#4683

### CODE REVIEW

File: src/field_specials.c [hunk 1]
```diff
@@ -311,51 +289,51 @@ u8 GetSSTidalLocation(s8 *mapGroup, s8 *mapNum, s16 *x, s16 *y)
...
+            *x = *varCruiseStepCount - 140;
+        }
+        break;
+    case SS_TIDAL_HALFWAY_SLATEPORT:
+        if (*varCruiseStepCount < 66)
+        {
+            *mapNum = MAP_NUM(ROUTE132);
+            *x = 65 - *varCruiseStepCount;
+        }
+        else if (*varCruiseStepCount < 146) {
```
Comment (LOuroboros) [inline src/field_specials.c:327]:
```suggestion
        else if (*varCruiseStepCount < 146)
        {
```

File: src/field_specials.c [hunk 2]
```diff
@@ -2713,13 +2590,14 @@ static void ScrollableMultichoice_ProcessInput(u8 taskId)
         {
             CloseScrollableMultichoice(taskId);
         }
-        // if selected option was the last one (Exit)
         else if (input == task->tNumItems - 1)
         {
+            // if selected option was the last one (Exit)
```
Comment (GriffinRichards) [inline src/field_specials.c:2595]: Drop "if" here, because it's known now

File: src/field_specials.c [hunk 3]
```diff
@@ -368,17 +347,15 @@ bool32 ShouldDoWallyCall(void)
...
-                    return FALSE;
-                }
-                break;
-            default:
+        case MAP_TYPE_TOWN:
+        case MAP_TYPE_CITY:
+        case MAP_TYPE_ROUTE:
+        case MAP_TYPE_OCEAN_ROUTE:
+            if (++(*GetVarPointer(VAR_WALLY_CALL_STEP_COUNTER)) < 250)
                 return FALSE;
```
Comment (kiliwily) [inline src/field_specials.c:355]: There are no parenthesis around this like in the other functions. Either they were forgotten or they shouldn't be in the other functions.

Comment (GriffinRichards) [reply src/field_specials.c:355]: The parenthesis format here is consistent with all the other `if (++(*GetVarPointer(VAR)) < NUM)` in this file. In any case this PR has been merged already, if you'd like to propose a change please open a PR or discuss it in the Discord

**Pull request merged**